app/views/application/_tag_update_preview.html.erb

```diff
@@ -1,10 +1,10 @@
 <div class="tag-update-preview">
   <br/>
 
-  <div data-module="tag-update-progress">
-    <% if tag_mappings.any? %>
+  <div data-module="import-progress">
```

Comment:
@Davidslv The standard way of loading Javascript modules in GOV.UK is to refer to them on a `data-module` attribute. The module is called `TagUpdateProgress`, so we need to keep this as `tag-update-progress`. More details [here](https://github.com/alphagov/govuk_admin_template/blob/master/JAVASCRIPT.md)